team-practices.md

```diff
@@ -0,0 +1,102 @@
+# FOIA Team Practices
+
+This document describes the conventions the FOIA team has agreed to work under.
+For information about licensing, code of conduct and contributing to this project,
+see the [CONTRIBUTING](CONTRIBUTING.md) file.
+
+## Sprints
+
+We do one-week sprints, with a retro and planning meeting every Monday, and a sprint review
```

Comment:
I thought we were doing 1 week sprints, that's what we did in discovery. I think Acquia was thinking two-week, but we need to clarify that. I'll bring it up in slack.